src/anm/media/video.js

```diff
@@ -47,7 +47,7 @@ Video.prototype.load = function(elm, player) {
     var me = this;
     ResMan.loadOrGet(player.id, me.url,
         function(notify_success, notify_error, notify_progress) { // loader
-            var url = me.url;
+            var url = Array.isArray(me.url) && me.url.length > 0 ? me.url[0] : me.url;
```

Comment:
Thanks! `Array.isArray(me.url) && (me.url.length > 0) ? me.url[0] : me.url` would be fine in this case, no need in extremes, of course.